tooling/config-eslint/utils/overrides.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    overrides: [
+        {
+            files: ['*.cjs', '*.js', '*.mjs'],
+            parser: '@typescript-eslint/parser',
```

Comment:
The `@vercel/style-guide` package uses [`@babel/eslint-parser`](https://www.npmjs.com/package/@babel/eslint-parser) for JS files by default, which isn't compatible with `@typescript-eslint/parser` & typed linting.